### ISSUE TITLE
Request for Dell Vostro 3471 EFI Configuration

### ISSUE THREAD

**Issue by Vturbo005** [therajuahammad/opencore-efi-i3-9100-coffee-lake-hackintosh#1]
Hello,

I have a Dell Vostro 3471 with the following specs:

Processor: Intel Core i3-9100
RAM: 8GB
Graphics: UHD 630
Would you be willing to help me adapt your EFI configuration for my system?

Thank you!

**Comment by therajuahammad** [comment]
yes, I think it’s suitable with your configuration. You can try with your own risk

**Comment by Vturbo005** [comment]
But my motherboard has the B365 chipset, and the Wi-Fi and Bluetooth adapters are Qualcomm then Can I use this efi for my pc
